src/app/components/videochat/jh-video-chat.html

```diff
@@ -132,7 +132,6 @@
 
 <script>
 $("#menu-toggle").click(function(e) {
-  e.preventDefault();
```

Comment:
I am not sure if removing this is the best thing to do, but I don't think it makes any difference.